PROTO_tests/tests/numeric_test.py

```diff
@@ -341,97 +402,88 @@ def test_arctan2(self, num_type, denom_type):
 
         assert np.allclose(
             [
-                np.arctan2(na_num[i], na_denom[i]) if truth_np[i] else na_num[i] / na_denom[i]
+                (
+                    np.arctan2(na_num[i], na_denom[i])
+                    if truth_np[i]
+                    else na_num[i] / na_denom[i]
+                )
```

Comment:
Can you check your black settings?  It still seems like these are being reformatted more than necessary.

Reply:
I've just written a script to use the correct line length in black.  That should fix this going forward.